Sources/ApexyURLSession/Helpers/URLSession+Concurrency.swift

```diff
@@ -20,7 +20,7 @@ extension URLSession {
     ///    - delegate: Delegate to get events about request (NOT WORKING💀)
     /// - Returns: Tuple with Data and URLResponse
     public func data(
-        for request: URLRequest,
+        forRequest request: URLRequest,
```

Comment:
This is a change in the public API, so I think you should update the major version in the `podspec` file. Or at least a minor version 😄

Reply:
we don't need this extension at all now. it was implemented at time when `URLSession` had their async/await methods ready for use only since **iOS 15**. so to make `URLSessionClient` implementataion for **iOS 13** and easy to read i have made this extension with async/await methods. 

now this methods are in `URLSession` available since **iOS 13** so you can just delete this file and it will work justs fine

Reply:
and also now you can get rid of `AsyncAwaitHelper` file too